### ISSUE TITLE
The performance of pre-trained model on ImageNet?

### ISSUE THREAD

**Issue by ycszen** [CSAILVision/semantic-segmentation-pytorch#94]
Could you offer the accuracy or error rate on the ImageNet of the pre-trained model? 
May these model will have better performance than the models offered by the official PyTorch?

**Comment by hangzhaomit** [comment]
Yes, they are better, but very close. I do not have the numbers at hand, but the difference is less than <1%.

**Comment by ycszen** [comment]
In your code, the first 7x7 conv is replaced by 3 3x3 conv. This will improve the performance obviously. Therefore, I want to know the accurate accuracy of your pre-trained resnet model. Thank you.
